articles/azure-functions/ip-addresses.md

```diff
@@ -76,15 +76,15 @@ For example, this is what the Western Europe JSON fragment might look like:
 
  For information about when this file is updated and when the IP addresses change, expand the **Details** section of the [Download Center page](https://www.microsoft.com/en-us/download/details.aspx?id=56519).
 
-## Inbound IP address changes
+## <a name="inbound-ip-address-changes"></a>Inbound IP address changes
 
 The inbound IP address **might** change when you:
 
 - Delete a function app and recreate it in a different resource group.
 - Delete the last function app in a resource group and region combination, and re-create it.
-- Delete an SSL binding, such as during [certificate renewal](../app-service/configure-ssl-certificate.md#renew-certificate)).
+- Delete an SSL binding, such as during [certificate renewal](../app-service/configure-ssl-certificate.md#renew-certificate).
 
-When your function app runs in a [Consumption plan](functions-scale.md#consumption-plan), the inbound IP address might also change when you haven't taken any actions such as the ones listed.
+When your function app runs in a [Consumption plan](functions-scale.md#consumption-plan), the inbound IP address might also change even when you haven't taken any actions such as the ones [listed above](#inbound-ip-address-changes).
```

Comment:
Not sure what the rule is for linking to anchors, but it made sense to link back up to the list, rather than just mention it.

Reply:
That works. Thanks @andresnava!